dynawo/sources/Models/CPP/ModelNetwork/DYNModelTwoWindingsTransformer.cpp

```diff
@@ -175,8 +175,7 @@ tapChangerIndex_(0) {
     modelPhaseChanger_->setHighStepIndex(phaseTapChanger->getNbTap() - 1. + lowIndex);
     modelPhaseChanger_->setCurrentStepIndex(phaseTapChanger->getCurrentPosition());
   } else if (ratioTapChanger) {
-    modelRatioChanger_.reset(new ModelRatioTapChanger(tfo->getID()));
-    modelRatioChanger_->setV1SupV2(vNom1_ > vNom2_);
+    modelRatioChanger_.reset(new ModelRatioTapChanger(tfo->getID(), tfo->getRatioTapChanger()->getTerminalRefSide()));
```

Comment:
As terminalRef is not compulsory into the iidm xsd scheme (minOccurs = 0), what happens if I don't get this attribute? What is returned by the getTerminalRefSide? Shouldn't it be catched into the if/else if loop into the CPP model (see above comment)?

Reply:
see #242